forge-gui/res/cardsfolder/t/the_doctors_childhood_barn.txt

```diff
@@ -0,0 +1,14 @@
+Name:The Doctor's Childhood Barn
+ManaCost:no cost
+Types:Plane Gallifrey
+R:Event$ Moved | ValidCard$ Creature | Destination$ Battlefield | ReplaceWith$ ETBTapped | ReplacementResult$ Updated | ActiveZones$ Command | Description$ Creatures enter the battlefield tapped.
+SVar:ETBTapped:DB$ Tap | ETB$ True | Defined$ ReplacedCard
+T:Mode$ ChaosEnsues | TriggerZones$ Command | Execute$ RolledChaos | TriggerDescription$ Whenever chaos ensues, for each opponent, choose up to one target nonland permanent that opponent controls. Untap those permanents. They phase out. They can't phase in for as long as CARDNAME remains face up. When a player planeswalks, those permanents phase in.
+SVar:RolledChaos:DB$ Untap | ValidTgts$ Permanent.nonLand+OppCtrl | TargetMin$ 0 | TargetMax$ OneEach | TargetsWithDifferentControllers$ True | TgtPrompt$ Select up to one target nonland permanent each opponent controls | SubAbility$ DBPhase
+SVar:OneEach:PlayerCountOpponents$Amount
+SVar:DBPhase:DB$ Phases | Defined$ Targeted | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | ConditionDefined$ Targeted | ConditionPresent$ Card | ConditionCompare$ GE1 | Triggers$ TrigPlaneswalk | StaticAbilities$ CantPhaseIn | RememberObjects$ Targeted | Duration$ Permanent | ForgetOnPhasedIn$ True
+SVar:CantPhaseIn:Mode$ CantPhaseIn | ValidCard$ Card.phasedOutIsRemembered | IsPresent$ Card.EffectSource+faceUp | PresentZone$ Command | Description$ They can't phase in for as long as EFFECTSOURCE remains face up. When a player planeswalks, those permanents phase in.
+SVar:TrigPlaneswalk:Mode$ PlaneswalkedFrom | ValidCard$ Card.EffectSource | Execute$ DBPhaseIn | TriggerZones$ Command | Secondary$ True | TriggerDescription$ When a player planeswalks, those permanents phase in.
```

Comment:
Using PlaneswalkedTo instead might be better?
Because of the one with `DontPlaneswalkAway$`